src/app/components/elements/PromoContent.tsx

```diff
@@ -4,13 +4,37 @@ import { apiHelper } from "../../services";
 import { Col, Row } from "reactstrap";
 import { IsaacContentValueOrChildren } from "../content/IsaacContentValueOrChildren";
 import { Tabs } from "./Tabs";
+import { Link } from "react-router-dom";
 
 export const PromoContent = ({ item }: { item: IsaacPodDTO }) => {
   const { title, subtitle, value, image, url, encoding } = item;
   const defaultImage = "/assets/ics_hero.svg";
   const path = image?.src
     ? apiHelper.determineImageUrl(image.src)
     : apiHelper.determineImageUrl(defaultImage);
+
+  interface LinkOrAnchorProps {
+    url?: string;
```

Comment:
Does this need to be optional?

Reply:
The difficulty here is that the IsaacPodDTO (which is the interface that the content uses when handed into PromoContent) has link as url as optional.  So I can make url non-optional, but then it would have to be string | undefined otherwise Typescript objects. Arguably that might be a better approach, so that if url is undefined it will still display {children} without a link element of any kind.